DataFormats/BTauReco/interface/ParticleTransformerAK4Features.h

```diff
@@ -0,0 +1,27 @@
+#ifndef DataFormats_BTauReco_ParticleTransformerAK4Features_h
+#define DataFormats_BTauReco_ParticleTransformerAK4Features_h
+
+//#include "DataFormats/BTauReco/interface/JetFeatures.h"
+#include "DataFormats/BTauReco/interface/SecondaryVertexFeatures.h"
+#include "DataFormats/BTauReco/interface/NeutralCandidateFeatures.h"
+#include "DataFormats/BTauReco/interface/ChargedCandidateFeatures.h"
+
+#include <vector>
+
+namespace btagbtvdeep {
+
+  class ParticleTransformerAK4Features {
+  public:
+    //JetFeatures jet_features;
+
+    std::vector<SecondaryVertexFeatures> sv_features;
+
+    std::vector<NeutralCandidateFeatures> n_pf_features;
+    std::vector<ChargedCandidateFeatures> c_pf_features;
+
+    //std::size_t npv;  // used by deep flavour
```

Comment:
Is this commented out code needed here? If not, please remove

Reply:
Same as for https://github.com/cms-sw/cmssw/pull/40706#discussion_r1099055626

And should be resolved with https://github.com/cms-sw/cmssw/pull/40706/commits/6727ef6e2ed76f0659c1b494adfeda38a53075e1